ambry-frontend/src/test/java/com.github.ambry.frontend/AmbryBlobStorageServiceTest.java

```diff
@@ -2073,12 +2073,12 @@ private void injectAccountAndContainerForPutAndVerify(Container container, boole
 
     Container legacyContainerForPublicBlob =
         new ContainerBuilder(Container.DEFAULT_PUBLIC_CONTAINER_ID, "containerForLegacyPublicPut",
-            Container.ContainerStatus.ACTIVE, "This is a container for putting legacy public blob", false, false, false,
-            false, refAccount.getId()).build();
+            Container.ContainerStatus.ACTIVE, "This is a container for putting legacy public blob",
+            refAccount.getId()).setCacheable(true).build();
     Container legacyContainerForPrivateBlob =
         new ContainerBuilder(Container.DEFAULT_PRIVATE_CONTAINER_ID, "containerForLegacyPrivatePut",
-            Container.ContainerStatus.ACTIVE, "This is a container for putting legacy private blob", false, false, true,
-            false, refAccount.getId()).build();
+            Container.ContainerStatus.ACTIVE, "This is a container for putting legacy private blob", refAccount.getId())
+            .build();
```

Comment:
looks like `setCacheable` is true in original code. Maybe `.setCacheable(true)`?

Reply:
Was incorrect since `cacheable` should be `false` in private containers